modules/power-monitoring-release-notes-0-1.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * power_monitoring/power-monitoring-release-notes.adoc
+
+:content-type: REFERENCE
+[id="power-monitoring-release-notes-0-1_{context}"]
+= Power monitoring Technology Preview - 0.1
```

Comment:
Oh yes, I absolutely agree, thank you for pointing that out!, otherwise, I would not notice that! This is my first big project and I am learning so much from it and from your review, so thank you very much! :tada: